### ISSUE TITLE
A11y_VSCode_VersionControl_Screenreader_screen reader does not announce information about operation.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#76977]
**Environment Details:**
VSCode Version : 1.36.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.3.1

**Repro Steps:**
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Publish branch to the remote Git repository by navigating to the bottom left pane, arriving at the cloud button next to the git branch and hit ENTER
5.Start Screen reader.
6. Observe screen reader announcement when click on  publish button.

**Actual:**
when user click on Commit all option or on the undo last commit option or on the publish button the screen reader does not announce status message of  performed action i.e. committing action undoing the last action or publishing the to git.

**Expected:**
screen reader should announce status message of whatever operation is performing in background

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader user will not get informed with the action performed.
 
**MAS Reference:**
MAS 4.1.3- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=1f363df4-1eaa-4680-9e0a-c63b09081427

**Attachment For Reference:** 
[nOT ANNOUNCING ANY THING.pptx](https://github.com/microsoft/vscode/files/3372973/nOT.ANNOUNCING.ANY.THING.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/76977
